poc/test_vec/04/Poplar1Aes128_0.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
There might be other breaking changes for VDAF-04, so we'll wait to update the test vectors until just before cutting.